src/components/Wizard/Wizard.tsx

```diff
@@ -1,22 +1,24 @@
-import React, { SyntheticEvent } from 'react';
+import React, { ComponentProps, SyntheticEvent, useCallback, useState } from 'react';
 import { Route, useLocation, useHistory } from 'react-router-dom';
 import { CustomFile, Modal, FileUploadProps } from '@amsterdam/bmi-component-library';
 import { Button } from '@amsterdam/asc-ui';
 import { ChevronLeft } from '@amsterdam/asc-assets';
 import { useDispatch, useSelector } from '../../store/CustomProvider';
 import { setFile, setMetadata, resetState, removeFileFromStore } from '../../store/dataSlice';
 import { getFileFromStore, getMetadataFromStore } from '../../store/selectors';
-import { FormProps } from './Step2';
 import Step1, { SupportedHTTPMethods } from './Step1';
-import Step2 from './Step2';
-import { PreviousButtonStyle, CancelButtonStyle } from './WizardStyles';
-import { MetadataGenericType } from '../../store/store';
-import appendTrailingSlash from '../../utils/appendTrailingSlash';
-import appendPathSegment from '../../utils/appendPathSegment';
+import { PreviousButtonStyle, CancelButtonStyle, ModalContentStyle, ModalTopBarStyle } from './WizardStyles';
```

Comment:
Ik vind het een vreemde naming voor components. Het is triviaal om binnen een CL of lib welke een styled/themed UI moet hebben, alles te gaan appenden met `Style`

Reply:
This convention was not invented by our team but by the CL team form which we adopted conventions, eslint rules, etc.

Reply:
Again, why adopt a bad practice or concept.

Reply:
Will create a separate PR for aligning ESLint with the new conventions